common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
I've verified on the Studio sandbox that things seem to work okay. Interestingly, you can see from the copyright headers that we are bundling up two different versions of JQuery UI:

http://studio-clytwynec.sandbox.edx.org/static/952cc48/js/factories/container.js

    /*! jQuery UI - v1.11.4 - 2015-09-10
    * http://jqueryui.com
    * Includes: core.js, widget.js, mouse.js, selectable.js
    * Copyright 2015 jQuery Foundation and other contributors; Licensed MIT */

    /*! jQuery UI - v1.10.0 - 2013-01-23
    * http://jqueryui.com
    * Includes: jquery.ui.core.js, jquery.ui.widget.js, jquery.ui.mouse.js, jquery.ui.position.js, jquery.ui.draggable.js, jquery.ui.droppable.js, jquery.ui.resizable.js, jquery.ui.selectable.js, jquery.ui.sortable.js, jquery.ui.accordion.js, jquery.ui.autocomplete.js, jquery.ui.button.js, jquery.ui.datepicker.js, jquery.ui.dialog.js, jquery.ui.menu.js, jquery.ui.progressbar.js, jquery.ui.slider.js, jquery.ui.spinner.js, jquery.ui.tabs.js, jquery.ui.tooltip.js, jquery.ui.effect.js, jquery.ui.effect-blind.js, jquery.ui.effect-bounce.js, jquery.ui.effect-clip.js, jquery.ui.effect-drop.js, jquery.ui.effect-explode.js, jquery.ui.effect-fade.js, jquery.ui.effect-fold.js, jquery.ui.effect-highlight.js, jquery.ui.effect-pulsate.js, jquery.ui.effect-scale.js, jquery.ui.effect-shake.js, jquery.ui.effect-slide.js, jquery.ui.effect-transfer.js
    * Copyright (c) 2013 jQuery Foundation and other contributors Licensed MIT */